classes/source/source_database.php

```diff
@@ -30,7 +30,7 @@
 
 defined('MOODLE_INTERNAL') || die;
 
-define('TOOL_ETL_MAX_RECORDS', 5000);
+define('TOOL_ETL_MAX_RECORDS', PHP_INT_MAX);
```

Comment:
Why do we need this limit at all?

Reply:
That is legacy from custom sql reports. They have setting to limit query results, so I did same way

Reply:
yeah! But I don't understand why we use it here?